### DISCUSSION TITLE
Repulsive force at short distances

### DISCUSSION THREAD

**Discussion by siamak-attarian** [ACEsuit/mace#348] [Q&A] [answered]
Hello,

I was wondering if there is some kind of a repulsive force (like ZBL type) at short distances implemented in the mace code?

**Comment by ilyes319** [comment, marked as answer]
Hey @siamak-attarian, we just implemented that, it is available in the develop branch. You just need to add the argument `--pair_repulsion` to your input script, and it wll add ZBL repulsion for 2 body on top of mace.
